simplematrixbotlib/match.py

```diff
@@ -41,6 +43,34 @@ def is_from_userid(self, userid):
         """
         return self.event.sender == userid
 
+    def is_from_allowed_user(self):
+        """
+        Returns
+        -------
+        boolean
+            Returns True if the event was sent from an allowed userid
+        """
+        allowlist = self._bot.config.allowlist
+        blocklist = self._bot.config.blocklist
+        sender = self.event.sender
+        # if there is no explicit allowlist, default to allow
+        is_allowed = False if len(allowlist) > 0 else True
```

Comment:
What about changing this line to the following?
```python
if len(allowlist) == 0:
     return True
```

Reply:
See the matrix room for more details.

Reply:
but that would return true without checking the `blocklist` which is definitely needed